detectors/TransLab/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-bullseye
+WORKDIR /detector
+
+COPY . .
+
+RUN apt-get update && apt-get install ffmpeg libsm6 libxext6  -y \
```

Comment:
Separate install requirements for python from apt requirements. Make a separate layer for apt requirements

Reply:
Also a separate layer for downloading weights for the neural network and installing the application itself